nBTC/tests/nbtc_tests.move

```diff
@@ -49,7 +52,9 @@ fun test_nbtc_mint() {
     let mut scenario = test_scenario::begin(sender);
     let ctx = scenario.ctx();
     let btc_treasury = x"509a651dd392e1bc125323f629b67d65cca3d4bb";
-    let (lc, mut cap) = init_nbtc(btc_treasury, ctx);
+    let (lc, mut cap) = init_nbtc(ctx);
+    cap.setup(lc.client_id().to_address(), FALLBACK_ADDR, btc_treasury);
```

Comment:
shall we move the setup call into `init_nbtc`?

Reply:
done!